corehq/apps/users/static/users/js/roles.js

```diff
@@ -225,6 +232,26 @@ hqDefine('users/js/roles',[
                         allowCheckboxId: null,
                         allowCheckboxPermission: null,
                     },
+                    {
+                        showOption: ko.computed(function () {
+                            return self.permissions.edit_web_users() || self.permissions.view_web_users();
+                        }),
+                        editPermission: self.permissions.edit_user_tableau_config,
+                        viewPermission: self.permissions.view_user_tableau_config,
+                        text: gettext("<strong>Manage Tableau Configuration</strong> &mdash; manage tableau configuration for web users"),
+                        showEditCheckbox: ko.computed(function () {
+                            return self.permissions.edit_web_users();
+                        }),
+                        editCheckboxLabel: "edit-user-tableau-config-checkbox",
+                        showViewCheckbox: true,
```

Comment:
I guess you can saw the view checkbox is still enabled but it's hidden since the parent container `showOption` is hidden ([here](https://github.com/dimagi/commcare-hq/blob/4b92dc891a0be748bdb36ae97e684724dc2396ee/corehq/apps/users/templates/users/partials/edit_role_modal.html#L47))